Fix test_nodereplacement_proactive by waiting for node deletion on AWS

am-agrawa opened the pull request:
fixes #7069

prsurve commented:
Test need's to be ran for AWS upi

prsurve commented on ocs_ci/ocs/node.py:
```diff
@@ -912,6 +912,7 @@ def delete_and_create_osd_node_aws_upi(osd_node_name):
     """
 
     osd_node = get_node_objs(node_names=[osd_node_name])[0]
+    log.info(f"OSD Node Object is {osd_node}")
```
why do we want to print object here ?

am-agrawa replied:
Used it for debugging, removed

amr1ta commented:
The changes are looking good only the log added to debug needs to be removed ,
 log.info(f"OSD Node Object is {osd_node}") 

But the last testrun had failed due to some command failure, can you please rerun.

am-agrawa commented:
> Test need's to be ran for AWS upi

Already ran- https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-trigger-test-pr/4183/

am-agrawa commented:
> The changes are looking good only the log added to debug needs to be removed , log.info(f"OSD Node Object is {osd_node}")
removed
> 
> But the last testrun had failed due to some command failure, can you please rerun.
It's failing on VMware, and will be taken as part of another issue. Added a comment above with same info.

am-agrawa commented:
> The changes are looking good only the log added to debug needs to be removed , log.info(f"OSD Node Object is {osd_node}")
> 
> But the last testrun had failed due to some command failure, can you please rerun.
Debug log is now removed.
It's failing on VMware, and will be taken as part of another issue. Added a comment above with same info.
